openedx/features/discounts/applicability.py

```diff
@@ -2,7 +2,13 @@
 """
 Contains code related to computing discount percentage
 and discount applicability.
+
+WARNING:
+Keep in mind that the code in this file only applies to discounts controlled in the lms like the first purchase offer,
+not most regular discounts like coupons or enterprise offers configured in ecommerce.
```

Comment:
Nitpicking, but coupons and enterprise offers are not necessarily more "normal", just different.

Reply:
i don't think i used the word normal here, but i can change the word regular to other?